packages/cli/src/prompts/prompt.ts

```diff
@@ -232,10 +207,17 @@ function contractMethods(
 ): any[] {
   const { contract: contractAlias, package: packageName } = fromContractFullName(contractFullName);
   const contractManager = new ContractManager(projectFile);
-  if (!contractManager.hasContract(packageName, contractAlias)) return [];
-  const contract = contractManager.getContractClass(packageName, contractAlias);
 
-  return contractMethodsFromAbi(contract, constant);
+  try {
```

Comment:
This may seem unrelated, but the reason this was changed is that now `deploy` can be used with contracts from _unlinked_ packages, and there won't be an implicit `add` as in `create`. Because of this, `contractManager.hasContract` is no longer reliable, and I thought it was better to just try to read the artifact from disk and catch the failure. I created the `ContractNotFound` custom error class to make this catch nicer, instead of matching on the error message.